README.md

```diff
@@ -6,6 +6,6 @@
 - [Risper](https://github.com/djava387)
 - [Airidas](https://github.com/Adaz99)
 - [Mehdi](https://github.com/LemonRiz)
```

Comment:
Can you add to the readme some information about what you have done and what will need to be updated?
  - Can just be bullet points of things that need to be changed.